setup.py

```diff
@@ -19,9 +19,10 @@
 import sys
 import six
 
+exec(open('ipopt/version.py').read())
```

Comment:
Ah, I see! I just have never seen this before, but since it seems to be common practice, I'm fine with it.